src/DynamoRevit/RevitPathResolver.cs

```diff
@@ -55,6 +55,19 @@ internal RevitPathResolver(string userDataFolder, string commonDataFolder)
             // Add the Revit_20xx folder for assembly resolution
             additionalResolutionPaths = new List<string> { currentAssemblyDir };
 
+			// Add Steel nodes to preload
+			var steelNodesDirectory = Path.Combine(currentAssemblyDir, "nodesSteelConnections", "steel-pkg", "bin");
```

Comment:
Is the format here broken or it's just a github display issue?

Reply:
i've had tabs instead of spaces. I've fixed it